tests/python/test_core.py

```diff
@@ -243,6 +243,14 @@ def test_init_with_ordination(self):
         # emperor is instantiated as needed but not yet setup
         self.assertTrue(isinstance(viz._emperor, Emperor))
 
+    def test_init_with_ordination_features(self):
+        viz = Empress(self.tree, self.table, self.sample_metadata,
+                      ordination=self.pcoa, shear_to_table=False)
```

Comment:
```suggestion
                      ordination=self.biplot, shear_to_table=False)
```
From some manual testing, it looks like the bug you discovered (thank you!) only appears when a biplot is passed as the ordination -- a normal ordination without features (like `self.pcoa`) doesn't seem to trigger this problem (but please correct me if I've misunderstood this).

Therefore I think it would make sense to use a biplot in this test.

Reply:
Yes, you're correct, thank you!!